website/content/docs/secrets/databases/index.mdx

```diff
@@ -259,6 +345,26 @@ address these challenges, you can configure one or more of the following options
   its passwords, and enables multiple clients using the same static role to
   transition slowly.
 
+</Tab>
+
+<Tab heading="GUI" group="gui2">
+
+![Partial screenshot of the Vault GUI showing the database level "Rotate static roles immediately" toggle](/img/gui/databases/databaseRotation.png)
+On the DB connection creation page, this toggle button will set the default value for static roles whether to rotate immediately or to skip the initial rotation if toggled off.
+
+![Partial screenshot of the Vault GUI showing the role level "Rotate immediately" toggle](/img/gui/databases/roleRotation.png)
+On the static role creation page, this toggle button will control whether this individual role password will be rotated immediately or skipped if toggled off.
+
+![Partial screenshot of the Vault GUI showing a provided password and toggled off Rotate immediately input](/img/gui/databases/rolePasswordNonRotate.png)
```

Comment:
```suggestion
```

Reply:
3 screenshots of what is essentially the same toggle is excessive. We should use screenshots sparingly. Both because they disrupt the ability to scan through instructional steps and because it increases the maintenance burden for the page